app/celery/scheduled_tasks.py

```diff
@@ -49,7 +56,13 @@
 )
 from app.notifications.process_notifications import send_notification_to_queue
 from app.v2.errors import JobIncompleteError
-from celery import Task
+from celery import Task, signals
+
+signals.after_task_publish.connect(xray_after_task_publish)
+signals.before_task_publish.connect(xray_before_task_publish)
+signals.task_failure.connect(xray_task_failure)
+signals.task_postrun.connect(xray_task_postrun)
+signals.task_prerun.connect(xray_task_prerun)
```

Comment:
This is super cool!

Are we doing any before/after perf tests to see if xray adds any latency to any celery stuff?

Reply:
Yep -- we will be looking at that after we confirm that this actually works :)

Reply:
the pains of not having a true 'dev' environment available at our whim :)